HW_1.py

```diff
@@ -0,0 +1,253 @@
+# Task 1
+import datetime
+
+class User:
+    def __init__(self, user):
+        self.user = user
+
+class Message(User):
+    def __init__(self, user, text):
+        self.text = text
+        self.user = user
+        self.datetime = datetime.datetime.now()
+
+    def show(self):
+        #self.datetime = datetime
+        print(f"[{self.datetime}] {self.user}: {self.text}")
+
+    def send(self):
+        return (f"[{self.datetime}] {self.user}: {self.text}")
```

Comment:
![image](https://user-images.githubusercontent.com/48160953/225771875-3e71a054-30de-4e1f-aed2-934d8bbf2363.png)

В данном методе **чат** никак не фигурирует, то есть мы посылаем сообщение не в чат, а куда-то в пустоту

```suggestion
    def send(self, chat):
        self.datetime = datetime.datetime.now()
        chat.recieve(self)
```

Reply:
В этом месте как раз уже можно задать атрибут `datetime`

Reply:
И ещё, не совсем понятно почему мы возвращаем строку, мы должны отправлять **сообщение** (`Message`), а не строки, списки, словари и т.д.